flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_ATSU.js

```diff
@@ -305,6 +305,10 @@ const addWaypointAsync = (fix, mcdu, routeIdent, via) => {
     }
 };
 
+const simBriefToMsfsNatFix = (simBriefFix) => {
+    return parseFloat(simBriefFix.pos_lat).toFixed(0).padStart(2, "0") + Math.abs(parseFloat(simBriefFix.pos_long)).toFixed(0).padStart(2, "0") + "N";
```

Comment:
```js
parseFloat(simBriefFix.pos_lat).toFixed(0)
```

why not use parseInt instead if we don't use the floats decimal anyways?

```js
("" + parseInt(simBriefFix.pos_lat)).padStart(2, "0")
```

Reply:
I can do that. I don't usually write js-isms, like concatenating an int to a string automatically casting it to a string, simply because I only write JS for this project, while all other projects are in different languages where things like that would be an error.

Reply:
Same more me, iirc I've read about `"" + ` being the fastest way to convert something to a string in js.